vtrace/tests/testbasic.py

```diff
@@ -59,6 +59,6 @@ def test_vtrace_mmaps(self):
 class VtraceBasicExecTest(VtraceBasicTest, vt_tests.VtraceExecTest):
     breakpoints = {
         'windows': 'ntdll.NtTerminateProcess',
-        'linux': 'ld.malloc',
+        'linux': 'libc.malloc',
```

Comment:
ld.malloc does not exist.  not sure what happened here.

Reply:
it used to :)